net-misc/youtube-viewer/youtube-viewer-3.1.9.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit gnome2-utils eutils perl-module vcs-snapshot
+
+DESCRIPTION="A command line utility for viewing youtube-videos in Mplayer"
+HOMEPAGE="http://trizen.googlecode.com"
+SRC_URI="https://github.com/trizen/youtube-viewer/tarball/${PV} -> ${P}.tar.gz"
+
+LICENSE="|| ( Artistic GPL-1+ )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="gtk"
+
+RDEPEND="
+	>=dev-lang/perl-5.16.0
+	dev-perl/Data-Dump
+	dev-perl/libwww-perl
+	|| ( media-video/mplayer[X,network]
+		media-video/mplayer2[X,network]
+		media-video/mpv[X] )
+	dev-perl/LWP-UserAgent-Cached
+	dev-perl/JSON-XS
+	virtual/perl-File-Spec
+	virtual/perl-Getopt-Long
+	virtual/perl-Term-ANSIColor
+	virtual/perl-Text-ParseWords
+	virtual/perl-Text-Tabs+Wrap
+	gtk? (
+		dev-perl/File-ShareDir
+		dev-perl/XML-Fast
+		>=dev-perl/gtk2-perl-1.244.0
+		virtual/freedesktop-icon-theme
+		x11-libs/gdk-pixbuf:2[X,jpeg]
+	)"
+DEPEND="dev-perl/Module-Build"
+
+SRC_TEST="do"
+
+src_prepare() {
+	perl-module_src_prepare
+}
+
+# build system installs files on "perl Build.PL" too
+# do all the work in src_install
+src_configure() { :; }
+src_compile() { :; }
+
+src_install() {
+	local myconf
+	if use gtk ; then
+		myconf="--gtk-youtube-viewer"
+	fi
+	perl-module_src_configure
+	perl-module_src_install
+
+	if use gtk ; then
+		domenu share/gtk-youtube-viewer.desktop
+		doicon share/icons/gtk-youtube-viewer.png
+	fi
+}
+
+pkg_preinst() {
+	use gtk && gnome2_icon_savelist
+	perl_set_version
+}
+
+pkg_postinst() {
+	use gtk && gnome2_icon_cache_update
+	einfo
+	elog "optional dependencies:"
+	elog "  dev-perl/LWP-Protocol-https or dev-perl/libwww-perl[ssl]"
```

Comment:
this is mandatory now https://github.com/trizen/youtube-viewer/blob/3.1.9/Build.PL#L52